safe/impact_function/multi_exposure_wrapper.py

```diff
@@ -966,13 +972,39 @@ def run(self):
 
             analysis_layers.append(impact_function.analysis_impacted)
             aggregation_layers.append(impact_function.aggregation_summary)
-            list_of_analysis_path.append(
-                full_layer_uri(impact_function.analysis_impacted))
+            exposure_key = (
+                impact_function.provenance['exposure_keywords']['exposure'])
+            dict_of_analysis_summary_path[exposure_key] = full_layer_uri(
+                impact_function.analysis_impacted)
+            dict_of_analysis_summary_id[exposure_key] = (
+                impact_function.analysis_impacted.id())
+
+            # eq raster on pop raster might not having this
+            if impact_function.exposure_summary:
+                dict_of_exposure_summary_path[exposure_key] = full_layer_uri(
+                    impact_function.exposure_summary)
+                dict_of_exposure_summary_id[exposure_key] = (
+                    impact_function.exposure_summary.id())
```

Comment:
I don't know if you care, but also the `exposure_breakdown` might not exist.

Reply:
what do you mean by `exposure_breakdown` sorry ?

Reply:
exposure summary table sorry: https://github.com/inasafe/inasafe/blob/develop/safe/definitions/layer_purposes.py#L161

The exposure breakdown by exposure type. If you have a census vector exposure or pop raster exposure, you won't have this layer. I think it's fine in your case